kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1282,6 +1282,15 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(0)
 
+  val ENGINE_SPARK_MAX_INITIAL_WAIT: ConfigEntry[Long] =
+    buildConf("kyuubi.session.engine.spark.max.initial.wait")
+      .doc("Max wait time for the initial connection to Spark engine. The engine will" +
+        " self-terminate no new incoming connection is established within this time." +
+        "This setting only applies at the CONNECTION share level.")
+      .version("1.8.0")
+      .timeConf
+      .createWithDefault(Duration.ofSeconds(0).toMillis)
```

Comment:
why disable if in default? 60s should be a good a default value as we discussed before

Reply:
Sure, let me fix it.